test/integration/controller/jobs/pod/suite_test.go

```diff
@@ -63,7 +63,10 @@ func TestAPIs(t *testing.T) {
 
 func managerSetup(opts ...jobframework.Option) framework.ManagerSetup {
 	return func(mgr manager.Manager, ctx context.Context) {
-		err := pod.SetupIndexes(ctx, mgr.GetFieldIndexer())
+		err := indexer.Setup(ctx, mgr.GetFieldIndexer())
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		err = pod.SetupIndexes(ctx, mgr.GetFieldIndexer())
```

Comment:
Doesn't this flaky happen in other integrations?

In other integrations, we don't set up indexer such as batch/job:

https://github.com/kubernetes-sigs/kueue/blob/0602e146fa8fd01bf83cb368acb027041ec1f011/test/integration/controller/jobs/job/suite_test.go#L62-L75

Reply:
Other suites don't use the core controllers and batch job controller.

Reply:
It makes sense. Thanks.